README.rst

```diff
@@ -76,13 +76,18 @@ In addition to the pure API implementation, this library features a number of hi
 make the development of bots easy and straightforward. These classes are contained in the
 ``telegram.ext`` submodule.
 
+After installing_ the library, be sure to check out the section on `working with PTB`_.
+
 Telegram API support
-====================
+~~~~~~~~~~~~~~~~~~~~
+
+All types and methods of the Telegram Bot API **7.7** are natively supported by this library.
+In addition, Bot API functionality not yet natively included can still be used as described `in our wiki <https://github.com/python-telegram-bot/python-telegram-bot/wiki/Bot-API-Forward-Compatibility>`_.
 
-All types and methods of the Telegram Bot API **7.7** are supported.
+.. _installing:
```

Comment:
is that needed for linking? should just work with headers no

oh I guess for furo? Can you comment this in if that is the case?

Reply:
you're right, it works without :)